chatmeleon-server/tsconfig.json

```diff
@@ -1,16 +1,14 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "esnext",
+    "module": "commonjs",
     "esModuleInterop": true, //This is required since some of the dependencies are CommonJS packages.
-    "moduleResolution": "NodeNext", // required,
+    "moduleResolution": "node", // required,
     "outDir": "./dist",
-    "rootDir": "./src",
+    "rootDir": "./",
     "strict": true
   },
-  "ts-node": {
-    "esm": true,
-    "experimentalSpecifierResolution": "node"
-  },
-  "include": ["src"]
-}
+  "include": [
+    "./"
+  ]
+}
```

Comment:
Same as above. Changes may be irrelevant to this PR. If required, keep them and fix the merge conflict later.